fern/docs.yml

```diff
@@ -528,6 +528,7 @@ navigation:
             contents:
               - page: Intercom
                 path: ./pages/docs/integrations/support/intercom.mdx
+                icon: fa-brands fa-intercom
           - page: Postman
```

Comment:
thoughts on using an astronaut icon here?

Reply:
assuming this is for postman